### PR TITLE
Package maintenance update

### LINKED ISSUE #594: RDT 1.3 Package Maintenance Updates
_This is a placeholder issue for overall package maintenance updates for the next release: `RDT 1.3.0`_

### PR DESCRIPTION
**Pull request by pvk-developer**
Resolves #594

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,29 +1,107 @@
...
+                      A "Data Transform Service" is a commercial offering
+                      that allows third parties (other than your employees and
+                      contractors) to access the functionality of the Licensed
+                      Work so that such third parties directly benefit from the
+                      reversible data transformation for data formatting, statistical
+                      processing, anonymization or contextual extraction features of
+                      the Licensed Work.
+
+Change Date:          Change date is four years from release date. Please see
+                      https://github.com/sdv-dev/RDT/releases
```
Comment (npatki) [inline LICENSE:23]: Nitpick: Original license take had the sentence "Please see ..." on a new line, right underneath the sentence "Change date is four years ..."

**Review by npatki — APPROVED** [review]
Small comment but LGTM! Thanks for finding all the extra files we need to change too (eg. __init__ file)

File: README.md [hunk 2]
```diff
@@ -23,52 +24,27 @@
...
-[Slack Logo]: https://github.com/sdv-dev/SDV/blob/master/docs/images/slack.png
-[Community]: https://bit.ly/sdv-slack-invite
-[Google Colab Logo]: https://github.com/sdv-dev/SDV/blob/master/docs/images/google_colab.png
-[Tutorials]: https://colab.research.google.com/drive/1T_3XSPPOVILATsyRV9xjQPa0hvM1vnM-?usp=sharing
+RDT (Reversible Data Transforms) is a Python library that transforms raw data into fully numerical
+data, ready for data science. The transforms are reversible, allowing you to convert from numerical
+data back into your original format.
 
-# Install
+<img align="center" src="https://github.com/sdv-dev/SDV/blob/master/docs/images/rdt_main_tranformation.png"></img>
```
Comment (amontanez24) [inline README.md:31]: why did we store the image on another repo?

**Review by amontanez24 — APPROVED** [review]
:shipit:

File: README.md (hunk 2 above)
Comment (amontanez24) [reply README.md:31]: nvm I see we did this before

**Pull request merged**